CuratedSyncExamples/demo/src/main/java/io/realm/curatedsyncexamples/fieldencryption/ui/login/LoginScreen.kt

```diff
@@ -112,23 +125,53 @@ fun LoginBox(
         )
 
         OutlinedTextField(
-            modifier = Modifier.padding(top = 16.dp),
+            modifier = Modifier
+                .padding(top = 16.dp)
+                .focusRequester(emailRef)
+                .onKeyEvent {
+                    if (it.nativeKeyEvent.keyCode == KeyEvent.KEYCODE_ENTER) {
+                        localFocusManager.moveFocus(FocusDirection.Next)
+                        true
+                    }
+                    false
+                },
             value = email,
             isError = state.errorMessage != null,
             enabled = !state.loggingIn,
-            onValueChange = { email = it },
+            singleLine = true,
+            onValueChange = { email = it.trim() },
             placeholder = { Text("your@email.com") },
             label = { Text("Email") },
+            keyboardOptions = KeyboardOptions(imeAction = ImeAction.Next)
         )
         OutlinedTextField(
-            modifier = Modifier.padding(top = 4.dp),
+            modifier = Modifier
+                .padding(top = 4.dp)
+                .focusRequester(passwordRef)
+                .onKeyEvent {
+                    if (it.nativeKeyEvent.keyCode == KeyEvent.KEYCODE_ENTER) {
+                        localFocusManager.moveFocus(FocusDirection.Next)
+                        true
+                    }
+                    false
+                },
```

Comment:
I think this could be an extension function like:

```
fun Modifier.focusNextOnEnter(): Modifier = onKeyEvent {
  if (it.nativeKeyEvent.keyCode == KeyEvent.KEYCODE_ENTER) {
      localFocusManager.moveFocus(FocusDirection.Next)
      true
  }
  false
}
```